### ISSUE TITLE
Can you give a quick hint re. the purpose of EqualLR?

### ISSUE THREAD

**Issue by daniel347x** [rosinality/style-based-gan-pytorch#90]
Greetings.  Without needing to go into detail, can you give a quick hint regarding the purpose of the `EqualLR` function class (near the top of `model.py`)?

No need to discuss the forward hook.

It seems that the weights are rescaled at every call to `forward`.  Am I correct?  Is this standard practice?  I have not seen it before.

Thanks!

**Comment by daniel347x** [comment]
I have found the topic area.  For others interested, search for "Equalized Learning Rate" on Google.
